### DISCUSSION TITLE
合作

### DISCUSSION THREAD

**Discussion by NeKoOuO** [hanydd/BilibiliSponsorBlock#7] [General]
我覺得不用自己造輪子 
可以跟[BiliRoamingX](https://github.com/BiliRoamingX)的作者 @zjns 合作
實現像 YouTube Revanced 那樣的效果
而且還有基礎用戶的保證 畢竟越多人用越好用

**Comment by hanydd** [comment]
好啊！如果能合作加入已有的产品那再好不过了。只是现在我手上没有安卓机，没有办法开发调试，可能要再等等…
（抱歉这么久才看到，之前一直没注意到discussion😂

**Comment by zjns** [comment]
Android 客户端真要实现的话，看了下，还有点麻烦，以后需求大了有机会再说吧

**Reply by buluoha** [reply]
蹲大佬们一起做出合体版

**Comment by zjns** [comment]
已完成，见：https://github.com/BiliRoamingX/BiliRoamingX-PreBuilds/releases/latest

**Reply by hanydd** [reply]
优秀！
